### ISSUE TITLE
https://sputnik.roen.us not working to download the install script

### ISSUE THREAD

**Issue by thefunkygibbon** [PiPass/blockpage#76]
is it possible to upload that install script to github too?

**Comment by roenw** [comment]
We are moving to a bash script based installation available at https://raw.githubusercontent.com/PiPass/bin/master/install.sh which is why the PHP script has been removed.

**Comment by thefunkygibbon** [comment]
Yeah I found that in the end. Used it but still can't get tit to work :(
